khmer/_khmer.cc

```diff
@@ -5215,6 +5215,499 @@ static PyObject * hllcounter_merge(khmer_KHLLCounter_Object * me,
     Py_RETURN_NONE;
 }
 
+/********************************
+ * Assembler classes
+ ********************************/
+
+
+typedef struct {
+    PyObject_HEAD
+    LinearAssembler * assembler;
+} khmer_KLinearAssembler_Object;
```

Comment:
For my education: do you know what the system behind using or not that extra K is (`khmer_KLinearAssembler_Object` vs `khmer_LinearAssembler_Object`)?